TenureInformationApi/V1/Controllers/TenureInformationController.cs

```diff
@@ -81,5 +89,62 @@ public async Task<IActionResult> UpdateTenureForPerson([FromRoute] UpdateTenureR
 
             return NoContent();
         }
+
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        [HttpPatch]
+        [Route("{id}")]
+        [LogCall(LogLevel.Information)]
+        public async Task<IActionResult> EditTenureDetails([FromRoute] TenureQueryRequest query, [FromBody] EditTenureDetailsRequestObject editTenureDetailsRequestObject)
+        {
+            // get raw body text (Only the parameters that need to be changed will be sent.
+            // Deserializing the request object makes it imposible to figure out if the requester
+            // wants to set a parameter to null, or to not update that value.
+            // The bodyText is the raw request object that will be used to determine this information).
+            var bodyText = await HttpContext.Request.GetRawBodyStringAsync().ConfigureAwait(false);
+
+            try
+            {
+                var tenure = await _editTenureDetailsUseCase.ExecuteAsync(query, editTenureDetailsRequestObject, bodyText).ConfigureAwait(false);
+
+                if (tenure == null) return NotFound();
+
+                return NoContent();
+            }
+            catch (EditTenureInformationValidationException e)
+            {
+                // either only tenureStartDate or tenureEndDate have been updated, but fail validation against items in database             
+
+                var response = BuildCustomEditTenureBadRequestResponse(e.ValidationResult);
+
+                return BadRequest(response); ;
+            }
+        }
+
+        private static CustomEditTenureDetailsBadRequestResponse BuildCustomEditTenureBadRequestResponse(ValidationResult validationResult)
+        {
+            var errorResponse = new Dictionary<string, List<string>>();
+
+            foreach (var error in validationResult.Errors)
+            {
+                // create list at key if it doesnt exist
+                if (!errorResponse.ContainsKey(error.PropertyName)) errorResponse.Add(error.PropertyName, new List<string>());
+
+                var errorObject = new
+                {
+                    ErrorCode = error.ErrorCode,
+                    ErrorMessage = error.ErrorMessage,
```

Comment:
To make this consistent with how "normal" validation errors get returned you need to add
`CustomState = error.CustomState`
(See the Infrastructure class `UseErrorCodeInterceptor`)